hubblestack/extmods/fdg/cert_discovery.py

```diff
@@ -0,0 +1,168 @@
+# -*- encoding: utf-8 -*-
+"""
+Flexible Data Gathering: cert_discovery
+=============================
+Intention -
+This fdg module allows connecting to open ports on a system and retrieving
+certificate details that might be attached on those ports.
+
+Testing -
+    1. Configure this module through Hubble's schedule using the following configuration
+        fdg_cert_discovery:
+        function: fdg.fdg
```

Comment:
Is indentation of this example block correct ?

Reply:
Fixed